cmd/kogito/command/completion/completion.go

```diff
@@ -27,8 +27,8 @@ type completionCommand struct {
 	Parent  *cobra.Command
 }
 
-// newCompletionCommand is the constructor for the completion command
-func newCompletionCommand(ctx *context.CommandContext, parent *cobra.Command) context.KogitoCommand {
+// initCompletionCommand is the constructor for the completion command
+func initCompletionCommand(ctx *context.CommandContext, parent *cobra.Command) context.KogitoCommand {
```

Comment:
do we need then to return the `context.KogitoCommand` if that is all happening in `InitHook` ?

Reply:
We need to return the command in case this command becomes parent of another.

And we need to call the `InitHook` nevertheless.

Reply:
ok